### ISSUE TITLE
Renaming a collection does not affect the graph

### ISSUE THREAD

**Issue by ghost** [arangodb/arangodb#2132]
## my environment running ArangoDB

I'm using the latest ArangoDB of the respective release series:
- [ ] 2.8
- [x] 3.0
- [ ] self-compiled devel branch

On this operating system:
- [ ] DCOS on
  - [ ] AWS
  - [ ] Azure
  - [ ] own infrastructure
- [x] Linux 
  - [ ] Debian .deb
  - [x] Ubuntu .deb
  - [ ] SUSE   .rpm
  - [ ] RedHat .rpm
  - [ ] Fedora .rpm
  - [ ] Gentoo
  - [ ] docker - official docker library
  - [ ] other:
- [ ] Windows, version: 
- [ ] MacOS, version:
### this is a web interface-related issue:

I'm using the web interface with this browser: Firefox/Chrome     running on this OS:
- [x] _system database?

These are the steps to reproduce:
1) open the browser on http://localhost:8529
2) log in
3) use database [x] `_system` [ ] other: 
4) click to collection 
5) click to settings
6) Rename the Collection

The following problem occurs: 

In my arangodb, version 3.0.10, there are two Collections: Entity and EntityRelation. The Entity contains only entities and the EntityRelation contains the relations between Entities. So, when renaming those collections from the **User Interface**, (i.e.: Entity -> EntitySecond and EntityRelation -> EntityRelationSecond) the graph does not display the relations between entities.

The fields, of each document of the Entity collection, are the following: 
_ 
_id: Entity/48083120
_rev: 48083120
_key: 48083120
other fields_

The fields, of each document of the EntityRelation collection, are the following: 
_ 
_id: EntityRelation/FROM221498TO285746TypesRELATION_SUBSIDIARY
_rev: 48326528
_key :FROM221498TO285746TypesRELATION_SUBSIDIARY
_from: Entity/48083120
_to: Entity/48273730
other fields_

In my opinion the issue of this is because:
When changing the name of the **Entity** Collection to **EntitySecond**, the field **_id** in **EntitySecond** changes to **"EntitySecond/48083120"**, as it should. And the field **_id** does the same for the **EntityRelationSecond**. But, the fields **_from** and **_to** from **EntityRelationSecond** do not change to "EntitySecond/48083120" and "EntitySecond/48273730", respectively.

So, when you create the graph for this collections, with the following options:
_Edge definitions: EntityRelationSecond
fromCollections: EntitySecond
toCollections: EntitySecond_
,as the collection EntityRelationSecond still has the attributes **_from:"Entity/48083120"** and **_to: "Entity/48273730"**, the relations between them point to the **old Entity collection** which after renaming it, does not exist.

**Comment by dothebart** [comment]
Hi, 
yes, thats (a bit odd) but intended behaviour. 
Since ArangoDB 3.0 Edges reference the vertex collections by name; before they used to reference them by collection id. This would i.e. put you into trouble when you drop & recreate the vertex collection (i.e. from an earlier dump; see #1581 for more details) 

So if you rename a vertex collections you have to adjust the edges pointing to the new name of the vertex collection. 

so an example picking https://docs.arangodb.com/3.2/Manual/Graphs/#the-knowsgraph and editing it could consist of these 2 steps:

- Rename the collection `persons` to `people` (as you already did)
- run an AQL to fix the edges:
```
FOR u IN knows 
  UPDATE u
     WITH {
         _from: REGEX_REPLACE(u._from, "persons/", "people/"),
         _to: REGEX_REPLACE(u._to, "persons/", "people/")}
    IN knows
```

As you can see you need to modify all edges in question; This could be resource intense depending on the size of your edge collection. You may want to add more `FILTER` statements, and introduce `LIMIT` so you don't need to change all documents at once. 

Thus I will mark this issue as won't fix and workaround available.

**Comment by loredra** [comment]
Tmy understanding,`UPDATE` cannot be done on system generated properties such as _from, to and `REGEX_REPLACE` is not available on v3.0.2. Is it correct?
